### ISSUE TITLE
datatype 'Contactinfo' uit applicatieprofiel ‘Generiek - basis’: eigenschappen met te beperkte kardinaliteit

### ISSUE THREAD

**Issue by jeroen-devlieghere** [Informatievlaanderen/OSLO-Discussion#409]
‘Contactinfo’ is een datatype dat ingesteld is in het [applicatieprofiel ‘Generiek - basis](https://data.vlaanderen.be/doc/applicatieprofiel/generiek-basis/#Contactinfo)’ en door veel andere OSLO-standaarden is overgenomen, zo ook in AP 'Feitelijke verenigingen'.

Bij de implementatie van deze standaard in het Verenigingsloket merken we dat de kardinaliteit van enkele eigenschappen van dit datatype te beperkend is om data uit de realiteit goed te kunnen modelleren. 

Zo krijgen de eigenschappen 'telefoon', 'mail' en 'website' de kardinaliteit 0...1  
De definitie van Contactpunt is "Informatie zoals email, telefoon, adres die toelaat om iemand of iets te contacteren", dit betekent o.i. dat als we een contactpunt voor een persoon instellen (bv.: "contact werk" of "contact thuis"), of voor een vereniging (bv. "contact ledenwerving", "contact bestuur") deze persoon of vereniging telkens maar max. één telefoon, e-mailadres of website kan opgeven. Terwijl personen of organisaties ook voor één doeleinde vaak over meerdere telefoonnummers, mailadressen... kunnen beschikken. Zo kan iemand privé te bereiken zijn op een mobiel nummer én op een vaste lijn. 

Kan dit eens herbekeken worden?

**Comment by GeertThijs** [comment]
Inderdaad. Bij Organisatie kan je dat nog omzeilen doordat de kardinaliteit van Organisatie.contactinfo 0..* is (omdat binnen eenzelfde Organisatie meerdere contactpunten kunnen zijn, bvb algemene dienst met info@organisatie.be en boekhouding met boekhouding@organisatie.be (hoewel je ook deze departementen apart zou kunnen beschrijven met elk hun eigen Contactinfo. Bij Persoon is geredeneerd dat je geen x Contactinfo's voor eenzelfde Persoon kan hebben. Maar uiteraard is het mogelijk dat de Persoon meerdere emails of telefoonnummers heeft. Makkelijkst is wellicht om alle attributen een kardinaliteit 0.;* mee te geven. de attributen zijn voldoende onafhankelijk van elkaar om dat te doen, bvb het is niet dat bij een andere email een ander telefoonnummer hoort normaal gezien...
